crates/coresimd/Cargo.toml

```diff
@@ -18,6 +18,9 @@ is-it-maintained-issue-resolution = { repository = "rust-lang-nursery/stdsimd" }
 is-it-maintained-open-issues = { repository = "rust-lang-nursery/stdsimd" }
 maintenance = { status = "experimental" }
 
+[dependencies]
+cfg-if = "0.1"
```

Comment:
For ease of integration into rust-lang/rust, could this macro be inlined? (It's actually quite a small macro)

Reply:
Sure, done.